pilot/pkg/xds/bench_test.go

```diff
@@ -560,22 +560,30 @@ func BenchmarkPushRequest(b *testing.B) {
 
 func makeCacheKey(n int) model.XdsCacheEntry {
 	ns := strconv.Itoa(n)
+
+	// 100 services
+	services := make([]*model.Service, 0, 100)
+	// 100 destinationrules
+	drs := make([]*config.Config, 0, 100)
+	for i := 0; i < 100; i++ {
```

Comment:
@howardjohn If i have 100 services here,  we need to store
 svc1->key
 svc2->key 
...